instrumentation/mybatis/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mybatis/MapperMethodRequest.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
```

Comment:
our current convention is to add the earliest supported version to package name, so here it should be `io.opentelemetry.javaagent.instrumentation.mybatis.v3_2`

Reply:
> our current convention is to add the earliest supported version to package name, so here it should be `io.opentelemetry.javaagent.instrumentation.mybatis.v3_2`

I adjusted it.